.github/workflows/create-demo-clusters.yml

```
@@ -255,6 +256,7 @@ jobs:
           flavor: gke-default
           name: ${{ inputs.cluster-with-real-load-name }}
           lifespan: 168h
+          description: "Long-running cluster (real load) for testing ${{ inputs.version }}"
```

```suggestion
          description: "Long-running cluster with real workload (with Collector and Berserker) for testing ${{ inputs.version }}"
```

Is it called berserker?